backend/src/main/java/com/github/clarakraus/susume/post/PostService.java

```diff
@@ -0,0 +1,23 @@
+package com.github.clarakraus.susume.post;
+
+import com.github.clarakraus.susume.blog.Movie;
+import com.github.clarakraus.susume.blog.MovieApiConnection;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PostService {
+
+    private final PostRepo postRepo;
+    private final MovieApiConnection movieApiConnection;
+
+    public void createPost(Post post) {
+        post.setCategory(Category.Movie);
+        postRepo.save(post);
+    }
+
+    public Movie getMovieById(long movieId) {
+        return movieApiConnection.getMovieFromTMDBById(movieId);
```

Comment:
Get movie by id, could be null or empty. So a try catch in service and controller would handle the case someone sends an invalid id.